core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/add-repositories.aarch64.sh

```diff
@@ -1 +1,13 @@
 #!/usr/bin/env bash -eu
+
+CENTOS_EXTRAS=$(cat <<'EOF'
```

Comment:
Do we need to add this repo?

It looks like standard OS repo and we have the following command in `download-requirements.sh` to enable it:
```
# -> CentOS-7 - Extras # for container-selinux and centos-release-scl packages
enable_repo 'extras'
```

Reply:
Without this repo there is a following error - https://stackoverflow.com/questions/65878769/cannot-install-docker-in-a-rhel-server